api/models/User.js

```diff
@@ -3,50 +3,20 @@ var promise       = require('promised-io/promise');
 var adBox         = require('boxfish-router');
 var colors        = require('colors');
 
+var customLogger = function customLogger() {
```

Comment:
@danieldmo want to merge this for now anyway? I'll create a jira for a logger repo